tools/base/src/Base.ts

```diff
@@ -84,18 +84,30 @@ export function SpectrumMixin<T extends Constructor<ReactiveElement>>(
             if (!activeElement) {
                 return false;
             }
+
             // Browsers without support for the `:focus-visible`
             // selector will throw on the following test (Safari, older things).
             // Some won't throw, but will be focusing item rather than the menu and
             // will rely on the polyfill to know whether focus is "visible" or not.
             try {
-                return (
+                return !!(
```

Comment:
`.matches` returns a boolean, I don't think we should need coercion here.

Reply:
Coercion is is necessary for `activeElement.shadowRoot?.activeElement?`, which could return `undefined`.